### DISCUSSION TITLE
Why are some trackers exempt?

### DISCUSSION THREAD

**Discussion by 88572** [zen-browser/desktop#228] [Q&A]
I was going to try out this browser, and I linked it in a Matrix chat. Someone pointed out that some trackers are exempt from tracker protection. Here's where it's happening: https://github.com/zen-browser/desktop/blob/1a450ebc727b2bfd5aa608b5816abbeba1618248/src/browser/app/profile/better-fox.js#L61
There are also some more exempt from it here: 
https://github.com/zen-browser/desktop/blob/1a450ebc727b2bfd5aa608b5816abbeba1618248/src/browser/app/profile/better-fox.js#L60
Why is this?

**Comment by mr-cheffy** [comment]
https://github.com/yokoffing/Betterfox/wiki/Optional-Hardening#block-embedded-social-posts-on-webpages

**Reply by mr-cheffy** [reply]
Allow embedded social media posts
